client/src/components/DataFiles/DataFilesProjectMembers/DataFilesProjectMembers.scss

```diff
@@ -10,6 +10,11 @@
   font-size: 12px;
 }
 
+
+.project-members__member-header {
+  color: #484848;
+}
+
```

Comment:
__Proposal__

Change this color by changing `.InfiniteScrollTable thead` color, instead.

Steps:

1. Change of `.InfiniteScrollTable thead` color from `#707070` to `var(--global-color-primary--x-dark)`\*
2. Removal of this `color: #484848;` style.
3. Removal of the then unnecessary `project-members__member-header` class.
4. (Consider) Removal of `InfiniteScrollTable`'s support for `headerClassName`.

> \* Because I have not yet found `#707070` used for table header color in any Adobe design. The options I found were:
> - `#484848` for header cell text of columns used to sort rows
> - `#484848` for header cell text of tables with no sorting (usually)‡
> - `#484848B3` (opacity) for header cell text of columns not used to sort rows of tables with sorting
> - `#7F7F7F` or `#484848B3` for header cell text of tables with no sorting (rarely)‡
>
> † Because `var(--global-color-primary--x-dark)` equals `#484848`.
> ‡ This is an inconsistency that is design's fault. The most common value may be assumed correct.

Reply:
**Clarification**

This will change some (hopefully all) table header cell colors from #707070 to #484848; this is a desirable change. I am okay with this. It's a step in the right direction.

> P.S. Any table header cells whose color does not change is likely adhering to a component-specific style for header cell text (a style which is most likely undesirable, and can be investigated when table styling is "fixed" app-wide).